src/EditorFeatures/Core/Implementation/EditorLayerExtensionManager.cs

```diff
@@ -83,6 +86,32 @@ public override void HandleException(object provider, Exception exception)
 
                 _errorLoggerService?.LogException(provider, exception);
             }
+
+            private void LogEnableAndIgnoreProvider(object provider)
+            {
+                if (IsRoslynCodefix(provider))
+                {
+                    Log(Infobar_EnableCodefixAndIgnoreFutureErrors, provider.GetType().Name);
```

Comment:
What all comes along for free with this? Hive, file version, etc.? /cc: @heejaechang